repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl

```diff
@@ -110,12 +110,14 @@
                                         </div><br>
                                     </#if>
 
+
                                     <#if landingPage??>
                                         <div class="heeuk-link-inactive-state-s">
-                                            <strong>Topics: </strong>
-
-                                            <#--  Publication professions  -->
-                                            <#if landingPage.publicationProfessions?has_content>
+                                            <strong>Publication Type: </strong>${landingPage.publicationType}
```

Comment:
1. It needs to retrieve value rather than key from publication type value list
2. I expect this should be a link rather than string , as it is a part of filters in publication collections

Reply:
Hi San, 

The changes are made to reflect the comments: 
- Publication type now show the value of the list
- Publication type is show as Link to the collection with filter